map/gps_tracker.cpp

```diff
@@ -111,3 +116,13 @@ void GpsTracker::OnLocationUpdated(location::GpsInfo const & info)
     return;
   m_track.AddPoint(info);
 }
+
+bool GpsTracker::IsRecentTrackCollectionInitialized() const
+{
+  return m_track.IsCollectionInit();
+}
+
+vector<location::GpsTrackInfo> GpsTracker::GetRecentTrackCollection() const
```

Comment:
```suggestion
std::vector<location::GpsTrackInfo> GpsTracker::GetRecentTrackCollection() const
```

Reply:
I remove the  `using namespace std; `

Reply:
@rtsisyk we use Unity builds which merge different cpp files into one large cpp file. So any `using namespace ...` directive in any one .cpp file leaks to other files and may cause build or link issues.

Reply:
@biodranik should we remove the `using namespace std...;` here?

Reply:
These refactorings can be done later. All new code should use std:: assuming that `using namespace ...` is _not_ present.